src/components/button/button.jsx

```diff
@@ -15,8 +16,22 @@ import styled from "styled-components";
  * @param onClick callback function to run when btn is clicked
  */
 
-// TODO integrate icons when all are catalogued
-// TODO update bg-opacity when theme color mapping done
+const StyledButton = styled.button`
+  border: none;
+  border-radius: 8px;
+  opacity: ${(props) => props.disabled && "0.5"};
+  width: ${(props) => props.size};
+  transition: opacity 0.2s;
+  &:hover {
+    opacity: ${(props) => !props.disabled && "0.8"};
+    cursor: ${(props) => props.disabled && "default"};
+  }
+`;
+
+const StyledText = styled.h3`
+  margin: 19px 0px;
```

Comment:
Could we make this margin: 20px 0 px; for consistency?  The other spacing elements are using the bootstrap spacings which are multiples of 4px.  The buttons are next to elements that have the 20px spacings.